packages/react/src/ReactElement.js

```diff
@@ -10,20 +10,15 @@ import {REACT_ELEMENT_TYPE} from 'shared/ReactSymbols';
 
 import ReactCurrentOwner from './ReactCurrentOwner';
 
-const hasOwnProperty = Object.prototype.hasOwnProperty;
+const objHasOwnProperty = Object.prototype.hasOwnProperty;
 
-const RESERVED_PROPS = {
```

Comment:
Related to #12368 #12370 #12530

This RESERVED_PROPS change longer required because Closure-compiler was fixed to not collapse properties from objects which are accessed using `hasOwnProperty`: https://github.com/google/closure-compiler/commit/c13cf48b98477e44409dba6359246bffa95b1c7b

This fix is included in release v20180319

Based on discussions on the mentioned PR's I think the change should be reverted.